lib/hammer_cli_foreman/user.rb

```diff
@@ -43,6 +43,7 @@ def extend_data(user)
         user["locale"] ||= _('default') if user.has_key?("locale")
         user["timezone"] ||= _('default') if user.has_key?("timezone")
         user["full_name"] = [user["firstname"], user["lastname"]].join(' ')
+        user["cached_usergroups"] = user["cached_usergroups"] - user["usergroups"]
```

Comment:
It doesn't work with `user.fetch('cached_usergroups',` [])` because I need get difference between all user groups and inherited user groups because of duplicity.